static/cache.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Cache</title>
+  <link rel="stylesheet" href="cache.css">
+</head>
+<body>
+  <h1>SCE TV Cache</h1>
+  <button id="playEntireDirectoryBtn">play entire directory</button>
+  <!-- Video -->
+  <div id="listOfVideosContainer" >
+  </div>
+  <script>
+          async function togglePlayVideo(e, video) {
+                  try {
+                    e.preventDefault();
+                    const file_path = video.path;
+                    const title = video.name;
+                    const thumbnail = video.thumbnail;
+                    let url = new URL("play/file", window.location.origin)
```

Comment:
```suggestion
                    let url = new URL("play/file", window.location.href)
```

Reply:
In the server.js, I defined the endpoint as "play/file", therefore if we use window.location.href, it would concatenate the current url to play/file, resulting in a call to /cache/play/file, which is not the url that I defined in the server.js. Do you want me to change the endpoint to cache/play/file?

Reply:
good point, its because we run the tv server at http://one.sce/tv, so we need to keep the `/tv` appended to the beginning of the url, but the below works?

<img width="350" alt="image" src="https://github.com/SCE-Development/sce-tv/assets/36345325/95ff47f5-aa31-4f2c-93a2-0b65f1d882d9">